php_phongo.c

```diff
@@ -1921,6 +1921,17 @@ static bool php_phongo_apply_wc_options_to_uri(mongoc_uri_t* uri, bson_t* option
 } /* }}} */
 
 #ifdef MONGOC_ENABLE_SSL
+
+/* This is copied from mongoc-ssl.c */
```

Comment:
Looking at [`_mongoc_ssl_opts_from_uri()`](https://github.com/mongodb/mongo-c-driver/blob/1.12.0/src/libmongoc/src/mongoc/mongoc-ssl.c#L97), I wonder if libmongoc also has a bug where it's copying these `const char *` into the `mongoc_ssl_opt_t` and later allowing them to be freed. I think the same fix I propose in https://github.com/mongodb/mongo-php-driver/pull/893/files#r204735780 to duplicate them first is warranted. If so, I'd suggest creating a CDRIVER ticket for that and opening a PR.

Reply:
Disregard. This wasn't a bug in libmongoc as their struct was stack-allocated.